android-client-sdk/src/main/java/com/devcycle/sdk/android/eventsource/DVCLifecycleCallbacks.kt

```diff
@@ -0,0 +1,59 @@
+package com.devcycle.sdk.android.eventsource
+
+import android.app.Activity
+import android.app.Application
+import android.os.Bundle
+import android.os.Handler
+import android.os.HandlerThread
+import com.devcycle.sdk.android.model.BucketedUserConfig
+import timber.log.Timber
+
+class DVCLifecycleCallbacks(
+    eventSource: EventSource,
+    onResume: () -> Any,
+    config: BucketedUserConfig? = null,
+    customHandler: Handler? = null
+) : Application.ActivityLifecycleCallbacks {
+    private val inactivityDelay: Long = config?.sse?.inactivityDelay?.toLong() ?: 800
+
+    private val eventSource = eventSource
+    private val onResume = onResume
+    private val listenerThread: HandlerThread = HandlerThread("DVCForegroundListener")
+    private val delayHandler: Handler
+    private var checkForegroundCallback = Runnable {
+        if(eventSource.state != ReadyState.CLOSED) {
```

Comment:
any reason to only check the closed state here but check the `closed` and `shutdown` states down below?

Reply:
nope just forgot it - fixed